### PR TITLE
Updated `test_landscape_scalable_bundle_db` test

### PR DESCRIPTION
**Pull request by TakoB222**
### Issue
User should be created on a relation-joined event, and the user should be dropped on a relation-departed event

### Solution
Added missed logic for `db_admin` test where checking that user dropped after relation removed

### CODE REVIEW

File: tests/integration/test_db_admin.py [hunk 1]
```diff
@@ -153,10 +158,27 @@ async def test_landscape_scalable_bundle_db(ops_test: OpsTest, charm: str) -> No
...
     # Remove the applications from the bundle.
     await ops_test.model.remove_application(LANDSCAPE_APP_NAME, block_until_done=True)
     await ops_test.model.remove_application(HAPROXY_APP_NAME, block_until_done=True)
     await ops_test.model.remove_application(RABBITMQ_APP_NAME, block_until_done=True)
 
+    # Remove the relation and test that its user was deleted
+    # (by checking that the connection string doesn't work anymore).
+    async with ops_test.fast_forward():
+        await ops_test.model.applications[DATABASE_APP_NAME].remove_relation(
+            f"{DATABASE_APP_NAME}:{RELATION_NAME}", f"{LANDSCAPE_APP_NAME}:{RELATION_NAME}"
```
Comment (dragomirp) [inline tests/integration/test_db_admin.py:175]:
```suggestion
           f"{DATABASE_APP_NAME}:{RELATION_NAME}", f"{LANDSCAPE_APP_NAME}:db"
```
The interface on Landscape's side is called `db`.

**Review by taurus-forever — APPROVED** [review]
Nice!

**Review by taurus-forever — COMMENTED** [review]
Vote back. Linter is not happy: https://github.com/canonical/postgresql-operator/actions/runs/8159411639/job/22388324477

File: tests/integration/test_db_admin.py (hunk 1 above)
Comment (dragomirp) [reply tests/integration/test_db_admin.py:175]: Relation name is `db-admin` but the interface on Landscape side is called `db`

**Review by marceloneppel — APPROVED** [review]

**Review by dragomirp — APPROVED** [review]

**Pull request merged**